geode-core/src/main/java/org/apache/geode/internal/logging/SortLogFile.java

```diff
@@ -36,9 +36,8 @@
  * This program sorts the entries in a GemFire log file (one written using a {@link LogWriter}) by
  * their timestamps. Note that in order to do so, we have to read the entire file into memory.
  *
- * @see MergeLogFiles
- * @see LogFileParser
- *
+ * @see org.apache.geode.internal.logging.MergeLogFiles
```

Comment:
Probably fully qualified from a move refactor

Reply:
I don't think either SortLogFile or MergeLogFiles moved so I'm not sure why IntelliJ made this change. Looks ok although I generally prefer to not add `@see` javadocs -- I usually just delete them.